api/system_info.go

```diff
@@ -0,0 +1,39 @@
+package api
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+)
+
+// SystemInfo returns information on the system and libpod configuration
+func (c *API) SystemInfo(ctx context.Context) (Info, error) {
+
+	var infoData Info
+
+	// the libpod/info endpoint seems to have some trouble
```

Comment:
```suggestion
	// The libpod/info endpoint seems to have some trouble
```

Reply:
Will remove this stale comment, we're not using the compat endpoint anymore.